client/gutenberg/extensions/contact-form/editor.scss

```diff
@@ -0,0 +1,144 @@
+
+.jetpack-contact-form {
+	padding: 10px 18px;
+
+	&.has-intro {
+		padding: 0;
+	}
+}
+
+.jetpack-contact-form .components-placeholder{
+	padding: 24px;
+
+	.components-placeholder__fieldset {
+		text-align: left;
+	}
+
+	.components-base-control {
+		margin-bottom: 16px;
+		width: 100%;
+	}
+}
+
+.jetpack-contact-form__intro-message {
+	margin: 0 0 16px;
+}
+
+.jetpack-contact-form__create {
+	width: 100%;
+}
+
+.jetpack-field-label {
+	display: flex;
+	flex-direction: row;
+
+	.components-base-control {
+		margin-bottom: -3px;
+
+		.components-form-toggle {
+			margin: 2px 8px 0 0;
+		}
+	}
+
+	.components-toggle-control .components-base-control__field {
+		margin-bottom: 0;
+	}
+}
+
+.jetpack-field-label__input {
+	flex-grow: 1;
+}
+
+// Duplicated to elevate specificity in order to overwrite core styles
+.jetpack-field-label__input.jetpack-field-label__input.jetpack-field-label__input {
+	border-color: #FFF;
+	border-radius: 0;
+	font-weight: 600;
+	margin: 0;
+	margin-bottom: 2px;
+
+	&:focus {
+		border-color: #FFF;
+		box-shadow: none;
+	}
+}
+
+.jetpack-field {
+	.components-text-control__input:disabled,
+	.components-textarea-control__input:disabled {
+		border-color: #8d96a0;
+		padding: 10px 8px;
+	}
+}
+
+// Duplicated to elevate specificity in order to overwrite core styles
+.jetpack-field-multiple__list.jetpack-field-multiple__list {
+	list-style-type: none;
+
+	&:empty {
+		display: none;
+	}
+
+	// TODO: make this a class, @enej
+	[data-type="jetpack/field-select"] & {
+		border: 1px solid #8d96a0;
```

Comment:
Just noting that eventually we'll need to look into moving hardcoded colours into variables, especially since some of these are Gutenberg-core colours. Won't need to happen in this PR though.

https://github.com/WordPress/gutenberg/blob/4dcf338ba603f47bf6027a764ab0b0d389e81700/assets/stylesheets/_colors.scss#L15

Reply:
Same with a lot of the other CSS properties and values as well. It's also always a good idea to start early.